app/decorators/concerns/program_decorator_common.rb

```diff
@@ -16,7 +16,7 @@ def to_values
           path = h.work_path(work)
           h.link_to(work.title, path, target: "_blank")
         when :started_at
-          send(field).to_time.strftime("%Y-%m-%d %H:%M")
+          send(field).to_time.strftime("%Y/%m/%d %H:%M")
```

Comment:
Do not use `to_time` on Date objects, because they know nothing about the time zone in use.